packages/esm-service-queues-app/src/clear-queue-entries-dialog/clear-queue-entries-dialog.resource.ts

```diff
@@ -25,10 +26,7 @@ export async function batchClearQueueEntries(queueEntries: Array<MappedVisitQueu
           },
           signal: abortController.signal,
           body: {
-            location: queueEntries[index]?.visitLocation,
-            startDatetime: parseDate(queueEntries[index]?.visitStartDateTime),
-            visitType: queueEntries[index]?.visitTypeUuid,
-            stopDatetime: new Date(),
+            stopDatetime: endedAt,
```

Comment:
This was an incidental discovery that t I ran across it as I was making changes to MappedVisitQueueEntry, and it appears to be a bug.  Basically, if you look at the code being removed in MappedVisitQueueEntry, it had a mapping as follows:  `visitStartDateTime: queueEntry.startedAt`

Then, in this part of the code, it is setting the `startDatetime` of the visit that is attached to each queueEntry being cleared from the queue to this mapped `visitStartDateTime`.  So if the queue entry being removed is from a later point in the visit than the very beginning, this will end up moving the visit start date to this later date and closing the visit.

In general, there doesn't seem to be any good reason for setting any property on visit here other than the stopDatetime, so I am removing them all.  If anyone can tell me why we should be doing this, we can discuss.

I am also ensuring that the endDate being set is defined once and set consistently across all queueEntries and visits, rather than setting an incrementally higher value as we iterate through the list.

FWIW, I think stopping the visit at all here is a mistake, and this is something we likely want to address in a subsequent ticket / discussion.

Reply:
Follow-up here.  I have raised a separate PR to address this particular issue, as it is a standalone and important bug that needs to be documented and prioritized, so this change will ultimately not be part of this PR.  Please find the separate PR here:  #1023